templates/rust/README.md.twig

```diff
@@ -0,0 +1,35 @@
+# {{ spec.title }} {{sdk.name}} SDK
+
+![License](https://img.shields.io/github/license/{{ sdk.gitUserName|url_encode }}/{{ sdk.gitRepoName|url_encode }}.svg?v=1)
+![Version](https://img.shields.io/badge/api%20version-{{ spec.version|url_encode }}-blue.svg?v=1)
+{% if sdk.warning %}
```

Comment:
would be nice if these if/endif blocks could be consistent with whitespace. I prefer:
```
{if}
thing..
{endif}```
without the whitespace between if & thing.